trezor_agent/gpg/protocol.py

```diff
@@ -47,9 +47,22 @@ def subpacket_byte(subpacket_type, value):
     return subpacket(subpacket_type, '>B', value)
 
 
+def subpacket_prefix_len(blob):
```

Comment:
I think that it's better to follow https://tools.ietf.org/html/rfc4880#section-4.2.2.
It is also the implementation used by GnuPG: [[1](https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=blob;f=g10/build-packet.c;h=86d42efe1828f1e6591ae2438c42ba44f07b4113;hb=refs/heads/master#l902), [2](https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=blob;f=g10/build-packet.c;h=86d42efe1828f1e6591ae2438c42ba44f07b4113;hb=refs/heads/master#l949)].